playbooks/06-deploy-edpm.yml

```diff
@@ -6,6 +6,12 @@
     step: pre_deploy
   ansible.builtin.import_playbook: ./hooks.yml
 
+- name: Deploy NFS server on target nodes
+  when:
+    - cifmw_edpm_deploy_nfs | default('false') | bool
+    - cifmw_architecture_scenario is undefined
```

Comment:
I think this may be useful also when an architecture is used

Reply:
maybe, I copied the conditions from the ceph one, removed it now